course.cabal

```diff
@@ -105,6 +105,7 @@ test-suite            tasty
 
   build-depends:
                       base < 5 && >= 3
+                      , containers >= 0.4.0.0
```

Comment:
`>= 0.4` would be better than having the redundant zeroes.

According to GHC/cabal's notion of a version, 0.4 and 0.4.0 and 0.4.0.0 are all valid versions, and the left ones are less than the right ones. So a bound of >= 0.4.0.0 won't allow 0.4.0 or 0.4.

It doesn't matter in this particular case, but it [really can make a difference sometimes](http://hackage.haskell.org/package/stratux-http-0.0.9/revisions/), so it's a good habit to get into.

Reply:
Interesting point @gwils . I just copy pasted what was in the build dependencies for the course.

Reply:
Fixed in master ;)

Reply:
No wukkas. At some point I should go through and straighten out all the version bounds in the course.